### PR TITLE
1.21 port

### PR DESCRIPTION
**Pull request by desht**
1.21

### CODE REVIEW

File: fabric/build.gradle [hunk 1]
```diff
@@ -3,8 +3,6 @@ plugins {
 	id "com.matthewprenger.cursegradle" version "1.4.0"
```
Comment (MichaelHillcox) [inline fabric/build.gradle]: You can yeet this now we use modmuss one :D

File: neoforge/build.gradle [hunk 2]
```diff
@@ -3,8 +3,6 @@ plugins {
 	id "com.matthewprenger.cursegradle" version "1.4.0"
```
Comment (MichaelHillcox) [inline neoforge/build.gradle]: Same here :D

**Review by MichaelHillcox — APPROVED** [review]
Just two very minor notes

File: fabric/build.gradle (hunk 1 above)
Comment (desht) [reply fabric/build.gradle]: Yeet!

File: neoforge/build.gradle (hunk 2 above)
Comment (desht) [reply neoforge/build.gradle]: Yeet!

**Review by MichaelHillcox — APPROVED** [review]

**Pull request merged**